### PR TITLE
Edit feed, part 2

### PR DESCRIPTION
**Pull request by caiosba**
## Description

Some more details for the "edit feed" page:

- [x] Add "View Shared Feed" button to the top
- [x] Show a confirmation modal when saving
- [x] Show warnings if there is no list or no list is selected

Reference: CV2-3128.

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Unit tests.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] I have removed the /* eslint-disable @calm/react-intl/missing-attribute */ from any files I've worked on and added a `description` prop to all `<FormattedMessage />` components that were missing it.
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

File: src/app/components/cds/buttons-checkboxes-chips/ButtonMain.js [hunk 1]
```diff
@@ -3,9 +3,11 @@ import PropTypes from 'prop-types';
 import Button from '@material-ui/core/Button';
 import styles from './ButtonMain.module.css';
 
+// FIXME: Refactor using native button instead of MUI Button
```
Comment (dariusk) [inline src/app/components/cds/buttons-checkboxes-chips/ButtonMain.js:6]: Do we have a ticket for this?

**Review by dariusk — APPROVED** [review]

**Review by amoedoamorim — APPROVED** [review]

**Comment by amoedoamorim** [comment]
:lgtm:

File: src/app/components/cds/buttons-checkboxes-chips/ButtonMain.js (hunk 1 above)
Comment (caiosba) [reply src/app/components/cds/buttons-checkboxes-chips/ButtonMain.js:6]: I will create one.

File: src/app/components/feed/SelectList.module.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.select-list-container {
```
Comment (dariusk) [inline src/app/components/feed/SelectList.module.css:1]: I added this and got rid of a `style` prop that was styling the `SelectList`. Should be more consistent with other components now.

**Review by dariusk — APPROVED** [review]
I tried to break this and couldn't! Seems like it's working as intended and looks good.

**Review by dariusk — APPROVED** [review]

**Pull request merged**